pkg/segment/search/searchaggs_test.go

```diff
@@ -445,3 +445,78 @@ func Test_PerformMeasureAggsOnRecs_WithList(t *testing.T) {
 		}
 	}
 }
+
+func getDummyNumericValueExpr(fieldName string) *structs.ValueExpr {
+	leftExpr := &structs.NumericExpr{
+		NumericExprMode: structs.NEMNumberField,
+		IsTerminal:      true,
+		ValueIsField:    true,
+		Value:           fieldName,
+		Op:              "",
+		Left:            nil,
+		Right:           nil,
+	}
+
+	rightExpr := &structs.NumericExpr{
+		NumericExprMode: structs.NEMNumber,
+		IsTerminal:      true,
+		ValueIsField:    false,
+		Value:           "2",
+		Op:              "",
+		Left:            nil,
+		Right:           nil,
+	}
+
+	expr := &structs.NumericExpr{
+		NumericExprMode: structs.NEMNumericExpr,
+		IsTerminal:      false,
+		ValueIsField:    false,
+		Value:           "",
+		Op:              "/",
+		Left:            leftExpr,
+		Right:           rightExpr,
+	}
+
+	return &structs.ValueExpr{
+		NumericExpr:   expr,
+		ValueExprMode: structs.VEMNumericExpr,
+	}
+}
+
+func getDummyMeasureAggregator(field string) *structs.MeasureAggregator {
+	return &structs.MeasureAggregator{
+		MeasureCol:      "",
+		MeasureFunc:     utils.List,
+		StrEnc:          "list(eval(" + field + "/2))",
+		ValueColRequest: getDummyNumericValueExpr(field),
+	}
```

Comment:
nit: you're calling these functions `getDummy...()` but it seems like they're coupled with each other and also with `Test_PerformMeasureAggsOnRecs_WithEvalWithField()`, which assumes they do `field / 2`. So it would be good to rename them to indicate they do `field / 2`

Reply:
updated.